tensorflow-logging/src/TensorFlow/Logging.hs

```diff
@@ -151,16 +155,27 @@ doubleWallTime = asDouble <$> getCurrentTime
 -- | Adds a 'CoreOps.histogramSummary' node. The tag argument is intentionally
 -- limited to a single value for simplicity.
 histogramSummary ::
-    (MonadBuild m, TensorType t, t /= ByteString, t /= Bool)
-     -- OneOf '[Int16, Int32, Int64, Int8, Word16, Word8, Double, Float] t)
+    (MonadBuild m, TensorType t, OneOf '[Int16, Int32, Int64, Int8, Word16, Word8, Double, Float] t)
```

Comment:
Why is this one preferable to the shorter above? (The comment was redundant, not sure why we keep it)

Reply:
I read the comment to indicate something wanting to be expressed, so I turned it into code instead. Agree it is shorter the way it was, but less explicit. So should I revert it and delete the comment?

Reply:
I think the intent is to allow any numeric tensortype, so the old way is somewhat more future proof assuming no more non-numeric types are added. I don't have a strong opinion on which way is better.